docs/keys.md

```diff
@@ -17,7 +15,7 @@ const ImageGallery = images =>
   </ul>
 ```
 
-Don't use an array index as key. If the position and number of items in a list is fixed, it will make no difference, but if the list is dynamic, the key will change every time the tree is rebuilt.
+Don't use an array index as key, if the index also specifies the order of siblings. If the position and number of items in a list is fixed, it will make no difference, but if the list is dynamic, the key will change every time the tree is rebuilt.
```

Comment:
@zaceno It took me a moment to understand this sentence. It makes sense now. Basically:

> Don't use an array index as key, if the index also specifies the order of items.

Do you think we'd lose preciseness, but gain clarity if we say "items" instead?